components/tidb_query/src/rpn_expr/impl_json.rs

```diff
@@ -14,6 +14,17 @@ fn json_type(arg: &Option<Json>) -> Result<Option<Bytes>> {
         .map(|json_arg| Bytes::from(json_arg.json_type())))
 }
 
+#[rpn_fn]
+#[inline]
+fn json_unquote(arg: &Option<Json>) -> Result<Option<Bytes>> {
+    arg.as_ref().map_or(Ok(None), |json_arg| {
+        json_arg
+            .unquote()
+            .map_err(|e| e.into())
```

Comment:
consider use `?` operator

Reply:
> consider use `?` operator

`json_arg` is in a clojure, `?` might cannot be called from inside a closure.

Reply:
```rust
#[rpn_fn]
#[inline]
fn json_unquote(arg: &Option<Json>) -> Result<Option<Bytes>> {
    arg.as_ref().map_or(Ok(None), |json_arg| {
        Ok(Some(Bytes::from(json_arg.unquote()?)))
    })
}
```

Reply:
> ```rust
> arg.as_ref().map_or(Ok(None), |json_arg| {
>         Ok(Some(Bytes::from(json_arg.unquote()?)))
>     })
> ```

You are right! I've done it.